tools/distpartitioning/dataset_utils.py

```diff
@@ -546,7 +546,11 @@ def get_dataset(
                     block_size=4096,
                     autogenerate_column_names=True,
                 )
-                parse_options = pyarrow.csv.ParseOptions(delimiter=" ")
+                parse_options = pyarrow.csv.ParseOptions(
```

Comment:
Can you add a unit test?

Reply:
Done. Modified the code accordingly in pytest_utils.py. This will make all the existing unit test cases to cover this code change. There is no need to add any additional unit test cases for this code change.

Reply:
I don't see any test case with etype_info[format][delimiter] set. Can you point me to the test case that covers this?

Reply:
@frozenbugs , In response to your earlier comment, about adding unit test case, I have made changes to the pytest_utils.py. In these changes, you will notice that we have a list of valid delimiters, which are " ', ",", ";", "\t". and we randomly chose from this list when we are running a given unit test case. Prior to this change, delimiter was always set to " ". 

Why there was no separate unit test case is added is because of the following reason. The way existing CI Test Framework works is as follows: 1. Create a input dataset, in the format which the pipeline can consume, 2. Invoke the pipeline on this input dataset, 3. After the pipeline is executed, CI Test cases validate the results obtained from running the graph partitioning pipeline. 

Now, in step 1), where input dataset is created I added code changes to use a random delimiter when edge files are created. Since step 1) is common for all the existing unit test cases, I did not add any specific unit test case. Basically I made sure that all the existing unit test cases are generated/executed in such a way that when edge files are created for their respective datasets random delimiter is used - as opposed to using a default delimiter. 

I hope you agree with this level of test coverage. Please suggest, if not. And I will appropriate instrument the proposed unit test case per your suggestion.

Reply:
unit test(s) for this code change will be merged with the following PR - https://github.com/dmlc/dgl/pull/5548